config/core/icons.json

```diff
@@ -61,5 +61,8 @@
   },
   "Construction": {
     "value": "construction"
+  },
+  "Logout": {
+    "value": "logout"
   }
```

Comment:
Yeah, I forgot to add this here and added it now.

Just to explain why it will also "works" when the icon is not added in the config:
This file is only required to generate the content of the Pattern-Library. The SVG-Sprite is not generated out of this file, but will use every `*.svg` file in the `public/assets/svgs` folder. This is part of the build step `build:svgs`

Reply:
Good to know :)